### PR TITLE
Parachain Staking (Collator selection / rewards) pallet

### PR DESCRIPTION
**Pull request by Wizdave97**

### CODE REVIEW

**Comment by drewstone** [comment]
Seems good, we need to add this to the parachain runtime and test!

**Comment by drewstone** [comment]
Can you also include this pallet in `dkg-substrate` repo? We will use it to test DKG over collators too.

**Comment by Wizdave97** [comment]
> Seems good, we need to add this to the parachain runtime and test!

Yeah, working on that

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -11,12 +11,7 @@ use arkworks_gadgets::{
 };
 use common::{AccountId, AuraId, Signature};
 use darkwebb_primitives::Balance;
-use darkwebb_runtime::{
-	wasm_binary_unwrap, AnchorVerifierConfig, AuraConfig, BLS381Poseidon3x5HasherConfig, BLS381Poseidon5x5HasherConfig,
-	BN254CircomPoseidon3x5HasherConfig, BN254Poseidon3x5HasherConfig, BN254Poseidon5x5HasherConfig, BalancesConfig,
-	CouncilConfig, GenesisConfig, MerkleTreeConfig, MixerVerifierConfig, ParachainStakingConfig, SudoConfig,
-	SystemConfig, UNITS,
-};
+use darkwebb_runtime::{AnchorVerifierConfig, AuraConfig, BLS381Poseidon3x5HasherConfig, BLS381Poseidon5x5HasherConfig, BN254CircomPoseidon3x5HasherConfig, BN254Poseidon3x5HasherConfig, BN254Poseidon5x5HasherConfig, BalancesConfig, CENTS, CouncilConfig, GenesisConfig, MerkleTreeConfig, MixerVerifierConfig, ParachainStakingConfig, SudoConfig, SystemConfig, UNITS, wasm_binary_unwrap};
```
Comment (drewstone) [inline node/src/chain_spec.rs:14]: spacing is off again

Comment (Wizdave97) [reply node/src/chain_spec.rs:14]: I ran `cargo fmt` before pushing, I'll do that again

**Pull request merged**